service/topology_coordinator.cc

```diff
@@ -1611,6 +1611,13 @@ class topology_coordinator : public endpoint_lifecycle_subscriber {
                         auto reason = ::format(
                             "bootstrap: insert tokens and CDC generation data (UUID: {})", gen_uuid);
                         co_await update_topology_state(std::move(guard), {std::move(mutation), builder.build()}, reason);
+
+                        co_await utils::get_local_injector().inject("stop_after_updating_cdc_generation",
```

Comment:
"stop" -> "pause"

Reply:
I see "topology_coordinator_pause_before_processing_backlog" injection in the same file, so changed it to "topology_coordinator_pause_after_updating_cdc_generation". But I'll keep "stop_after_updating_cdc_generation" as a name for error injection in the matrix.